test/unit_test/ccb_parser_test.cc

```diff
@@ -0,0 +1,143 @@
+#define BOOST_TEST_DYN_LINK
+
+#include <boost/test/unit_test.hpp>
+#include <boost/test/test_tools.hpp>
+
+#include "test_common.h"
+
+#include <vector>
+#include "conditional_contextual_bandit.h"
+#include "parser.h"
+
+CCB::label parse_label(parser* p, std::string label)
```

Comment:
Should this label parser be defined here? 

It seems like we would want to be testing the ccb_label_parser from ccb_label.h/ccb_label.cc

Reply:
This is correct, it is a test helper method.